src/core/hard_forks.zig

```diff
@@ -0,0 +1,85 @@
+const std = @import("std");
+const ArrayList = std.ArrayList;
+const allocator = std.mem.Allocator;
+const Slot = @import("slot.zig").Slot;
+pub const HardFork = struct { Slot, usize };
```

Comment:
can you add names to this struct and then reference the fields instead of [0] (better for readability)